src/libhirte/common/cfg.c

```diff
@@ -8,10 +8,11 @@
 #include <string.h>
 #include <unistd.h>
 
+#include <hashmap.h>
+
+#include "ini.h"
```

Comment:
Why is `ini.h` included as a local header (i.e. with "")  while `hashmap.h` is included as a system one (i.e. with <>)?